releases/efs-provisioner.yaml

```diff
@@ -0,0 +1,66 @@
+repositories:
+# Stable repo of official helm charts
+- name: "stable"
+  url: "https://kubernetes-charts.storage.googleapis.com"
+
+releases:
+
+################################################################################
+## EFS provisioner #############################################################
+################################################################################
+
+#
+# References:
+#   - https://github.com/helm/charts/blob/master/stable/efs-provisioner/values.yaml
+#   - https://github.com/kubernetes-incubator/external-storage/tree/master/aws/efs
+#
+- name: "efs-provisioner"
+  namespace: "codefresh"
```

Comment:
why `codefresh` namespace?
it could be deployed for any other purposes besides Codefresh

Reply:
should be `kube-system`